environment/src/main/scala/ru/primetalk/funtik/environment/geom2d/Vector2d.scala

```diff
@@ -0,0 +1,56 @@
+package ru.primetalk.funtik.environment.geom2d
+
```

Comment:
In mathematics it is usual to use short names.
`v` - is a normal name for a type that is called `V`. 
`p` - this might be replaced with something else. Do you have any suggestions?
`k` - it's a normal name for a scale coefficient.
`_1`, `_2` - got it. I used to these names and `x`, `y` are provided as well. Let's have some diversity ;)
`r` - is a mathematical name for one of the [polar coordinates](https://en.wikipedia.org/wiki/Polar_coordinate_system#Converting_between_polar_and_Cartesian_coordinates)

Reply:
```scala
    /** Rotates as a multiplication of complex numbers. */
    def rotate(o: V): V =
      vector2d(v._1 * o._1 - v._2 * o._2, v._1 * o._2 + v._2 * o._1)
```
of we replace `v` with `vector` and `o` with `other`:
```scala
    /** Rotates as a multiplication of complex numbers. */
    def rotate(other: V): V =
      vector2d(vector._1 * other._1 - vector._2 * other._2, vector._1 * other._2 + vector._2 * other._1)
```

I don't think that this helps in any way.